api/client/project.go

```diff
@@ -110,3 +110,72 @@ func (p *Padl) ListProjects() (*payloads.ListProjectsResponse, error) {
 
 	return &listProjResp, nil
 }
+
+// AddUser TODO
+func (p *Padl) AddUser(name string, email string, level int) (string, error) {
```

Comment:
call these AddUserToProject and RemoveUserFromProject. also, rename 'name' to 'project' or 'project name' and 'level' to 'priv' or 'privilegeLvl' or something